gpep517/__main__.py

```diff
@@ -84,12 +85,18 @@ def override_writestr(self, zinfo_or_arcname, data,
 
 
 @contextlib.contextmanager
-def patch_sysconfig(sysroot: str):
+def patch_sysconfig(sysroot: Path,
+                    prefix_from: typing.Optional[str],
+                    prefix_to: typing.Optional[str],
+                    ):
     stdlib_path = Path(sysconfig.get_path("stdlib"))
     if not stdlib_path.is_absolute():
         raise RuntimeError(f"stdlib path {stdlib_path} is not absolute")
-    data_paths = list(
-        (sysroot / stdlib_path.relative_to("/")).glob("_sysconfigdata_*.py"))
+    prefixed_stdlib = (Path(prefix_to or "/") /
+                       stdlib_path.relative_to(prefix_from or "/"))
+    prefixed_stdlib = sysroot / prefixed_stdlib.relative_to("/")
```

Comment:
I meant keeping this `sysroot_stdlib`, so that every variable would be self-explanatory.

Reply:
Right, fixed.